LICENSE.txt

```diff
@@ -0,0 +1,26 @@
+Copyright (c) 2017, Red Hat, Inc.
+
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without modification,
+are permitted provided that the following conditions are met:
+
+* Redistributions of source code must retain the above copyright notice, this
+list of conditions and the following disclaimer.
+* Redistributions in binary form must reproduce the above copyright notice,
+this list of conditions and the following disclaimer in the documentation
+and/or other materials provided with the distribution.
+* Neither the name of the <ORGANIZATION> nor the names of its contributors
+may be used to endorse or promote products derived from this software
+without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE REGENTS AND CONTRIBUTORS ''AS IS'' AND ANY
+EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL THE REGENTS OR CONTRIBUTORS BE LIABLE FOR ANY
+DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+(INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```

Comment:
This license text is subtly different from the text found at http://repoze.org/license.html, with the Repoze license carrying an additional condition than is included above, and some punctuation and wording differences as well. It also seems that we are not following the fourth condition of the repoze license :(

Reply:
Then, the question is which license fits the best because BSD license is mentioned in specfile.

If you need to discuss it further, I can just remove commit with license file which can be fixed later without blocking a new release.

Reply:
I think the Repoze license would be correct to use, because this project is a fork of https://github.com/tomlikestorock/pyramid_openid and his [README.txt](https://github.com/tomlikestorock/pyramid_openid/blob/master/README.txt) says "This code is offered under the BSD-derived Repoze Public License." He also mentions in there that he copied code from a now 404 URL that was licensed this way.

I think it's actually OK that the spec file says BSD, because I think this is considered to be the "modification variant" of the BSD, and Fedora's licensing guidelines have that as "BSD" still:

https://fedoraproject.org/wiki/Licensing:BSD#Modification_Variant

I suggest that we put the repoze license text that I linked above here, and that we send a PR to @tomlikestorock that adds the same to his repo as well. I'd be happy to do the latter if you want.

Reply:
Oh, and our fork of the project still has the text about Repoze in the README.txt as well.

Reply:
I've changed a content of the license file. PR for the orginal project is here: https://github.com/tomlikestorock/pyramid_openid/pull/10